cli/lib/cli/ide/gogland_user_pref_dir.rb

```diff
@@ -0,0 +1,15 @@
+require "cli/ide/jet_brains_ide_user_pref_dir"
+
+module Cli
+  module Ide
+    class GoglandUserPrefDir < JetBrainsIdeUserPrefDir
+      def ide_pref_dir_name_without_version
+        "Gogland"
+      end
+
+      def default_ide_pref_dir_version
+        "whatever"
```

Comment:
@moonmaster9000 

We noticed that the value of default_ide_pref_dir_version did not matter. We ran the ruby code and it detected 1.0 automatically. In the past, I've set this explicitly to the IDE_VERSION. It's been unclear how to handle this value when the IDE changes version numbers. For us, "whatever" more clearly aligns with the behavior we are seeing from the code. Any thoughts?